app/views/layouts/application.html.erb

```diff
@@ -9,6 +9,10 @@
   </head>
 
   <body>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery-datetimepicker/2.5.4/build/jquery.datetimepicker.full.min.js"></script>
+
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/jquery-datetimepicker/2.5.4/build/jquery.datetimepicker.min.css" />
+    <link href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css" rel="stylesheet">
```

Comment:
Thank you very much for all the feedback, I'll get to fixing it!
Yeah, I actually tried putting it in the head tag but it wouldn't load for some reason. Maybe I did something wrong, I'll try again.